server/routes/search.py

```diff
@@ -45,8 +46,10 @@ def search_dc():
     else:
         search_response = {}
 
-    # Convert from search results to template dictionary.
     results = []
+    results.append(ai.search(query_text))
```

Comment:
Can create a new route like '/search_nlp' which only has result from this?

Reply:
Done.